packages/vm/test/t8n/t8ntool.ts

```diff
@@ -218,10 +253,10 @@ export class TransitionTool {
     if (args.log === true) {
       this.vm.events.on('beforeTx', (_, resolve) => {
         // eslint-disable-next-line no-console
-        console.log('Processing new transaction...')
+        if (args.log === true) console.log('Processing new transaction...')
```

Comment:
This is wrapped inside an `if (args.log === true)` and I don't think the log flag can change, so I believe this is not necessary?